app/modules/individuals/parameters.py

```diff
@@ -26,29 +29,67 @@ class Meta(schemas.DetailedIndividualSchema.Meta):
 
 class PatchIndividualDetailsParameters(PatchJSONParameters):
     # pylint: disable=abstract-method,missing-docstring
-    OPERATION_CHOICES = (PatchJSONParameters.OP_REPLACE,)
+    OPERATION_CHOICES = (
+        PatchJSONParameters.OP_REPLACE,
+        PatchJSONParameters.OP_ADD,
+        PatchJSONParameters.OP_REMOVE,
+    )
+
+    # Is this weird? I need to modify EDM and Houston records
+    PATH_CHOICES_HOUSTON = ['encounters']
+
+    PATH_CHOICES_EDM = ['encounters']
+
+    PATH_CHOICES = tuple(
+        '/%s' % field for field in (PATH_CHOICES_EDM + PATH_CHOICES_HOUSTON)
+    )
+
+    @classmethod
+    def remove(cls, obj, field, value, state):
+        has_permission = rules.ObjectActionRule(obj, AccessOperation.WRITE).check()
+        ret_val = False
+        if has_permission:
+            if field == 'encounters':
+                for encounter_guid in value:
+                    if is_valid_uuid(encounter_guid):
+                        from app.modules.encounters.models import Encounter
+
+                        encounter = Encounter.query.filter(
+                            Encounter.guid == encounter_guid
+                        ).first()
+                        if encounter is not None and encounter in obj.encounters:
+                            obj.remove_encounter(encounter)
+                            ret_val = True
+            else:
```

Comment:
as '/encountes' is the only valid option, what's this supposed to be doing?

Reply:
Wha.. what if there are more options some day? Jk, removed.